takwimu/templates/takwimu/_includes/sections/enquire.html

```diff
@@ -0,0 +1,72 @@
+                    <label for="location">Your location</label>
+                    <select id="location" class="form-control">
+                        <option selected>Choose...</option>
+                        <option>...</option>
+                    </select>
+                </div>
+            </div>
+            <div class="form-group">
+                <label for="role">What best describe your role ?</label>
+                <select id="role" class="form-control">
+                    <option selected>Magician</option>
+                    <option>gynecologist</option>
+                </select>
+            </div>
+            <label for="services">What service(s) intrest you ?</label>
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <input class="form-check-input" type="checkbox" id="gridCheck">
+                    <label class="form-check-label" for="gridCheck">
+                        Service name number one
+                    </label>
+                </div>
+                <div class="form-group col-md-4">
+                    <input class="form-check-input" type="checkbox" id="gridCheck">
+                    <label class="form-check-label" for="gridCheck">
+                        Service name number 2
+                    </label>
+                </div>
+                <div class="form-group col-md-4">
+                    <input class="form-check-input" type="checkbox" id="gridCheck">
+                    <label class="form-check-label" for="gridCheck">
+                        Service name number 3
+                    </label>
+                </div>
+            </div>
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <input class="form-check-input" type="checkbox" id="gridCheck">
+                    <label class="form-check-label" for="gridCheck">
+                        Service name number 4
+                    </label>
+                </div>
+                <div class="form-group col-md-4">
+                    <input class="form-check-input" type="checkbox" id="gridCheck">
+                    <label class="form-check-label" for="gridCheck">
+                        Service name number 5
+                    </label>
+                </div>
+                <div class="form-group col-md-4">
+                    <input class="form-check-input" type="checkbox" id="gridCheck">
+                    <label class="form-check-label" for="gridCheck">
+                        Service name number 6
+                    </label>
+                </div>
+            </div>
+            <div class="form-group">
+                <label for="help">How can we help you?</label>
+                <textarea class="form-control" id="help" rows="3"></textarea>
+            </div>
+            <button type="submit" class="btn btn-primary float-right">Submit Enquiry</button>
+        </form>
+    </div>
+</section>
```

Comment:
Section needs to be in a `container`, and remove `col-8` for it first inner v

Reply:
cc @mtuchi